src/components/ClusterManager.js

```diff
@@ -279,72 +294,92 @@ export default class ClusterManager extends PureComponent {
 
     const app = currentApp(apps)
 
-    return div({ style: styles.container }, [
-      app && h(Clickable, {
-        style: { display: 'flex', marginRight: '2rem' },
-        onClick: () => {
-          // TODO galaxy SATURN-1855: implement this action
-        }
-      }, [
-        img({ src: galaxyLogo, alt: '', style: { marginRight: '0.25rem' } }),
-        div([
-          div({ style: { fontSize: 12, fontWeight: 'bold' } }, ['Galaxy']),
-          div({ style: { fontSize: 10, textTransform: 'uppercase' } }, [app.status])
-        ])
-      ]),
-      (activeClusters.length > 1 || activeDisks.length > 1) && h(Link, {
-        style: { marginRight: '1rem' },
-        href: Nav.getLink('clusters'),
-        tooltip: 'Multiple cloud environments found in this billing project. Click to select which to delete.'
-      }, [icon('warning-standard', { size: 24, style: { color: colors.danger() } })]),
-      h(Link, {
-        href: appLaunchLink,
-        onClick: window.location.hash === appLaunchLink && currentStatus === 'Stopped' ? () => this.startCluster() : undefined,
-        tooltip: canCompute ? `Open ${appName}` : noCompute,
-        'aria-label': `Open ${appName}`,
-        disabled: !canCompute,
-        style: { marginRight: '2rem', ...styles.verticalCenter },
-        ...(isRStudioImage ? {} : Utils.newTabLinkProps)
-      }, [isRStudioImage ? img({ src: rLogo, alt: '', style: { maxWidth: 24, maxHeight: 24 } }) : icon('terminal', { size: 24 })]),
-      renderIcon(),
-      h(IdContainer, [id => h(Fragment, [
-        h(Clickable, {
-          id,
-          style: styles.button(isDisabled),
+    return h(Fragment, [
+      div({ style: { ...styles.container, borderRadius: 5, marginRight: '1.5rem' } }, [
+        app && h(Clickable, {
+          style: { display: 'flex' },
+          disabled: appIsSettingUp(app) || appIsDeleting(app),
           tooltip: Utils.cond(
-            [!canCompute, () => noCompute],
-            [creating, () => 'Your environment is being created'],
-            () => 'Update cloud environment'
-          ),
-          onClick: () => this.setState({ createModalDrawerOpen: true }),
-          disabled: isDisabled
+            [appIsSettingUp(app), () => 'Your Galaxy application is being created'],
+            [appIsDeleting(app), () => 'Your Galaxy application is deleting'],
```

Comment:
Nit, for language, I think we should either say both:
`Your Galaxy application is being created` and `Your Galaxy application is being deleted` or `creating` and `deleting`

Reply:
Filtered out 'deleting' and 'predeleting' apps so removed the tool tip about deleting entirely.